apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/parser/graphql/GraphQLDocumentSourceBuilder.kt

```diff
@@ -89,7 +90,7 @@ object GraphQLDocumentSourceBuilder {
     get() {
       var indent = 0
       return lines().joinToString(separator = "\n") { line ->
-        if (line.endsWith("}")) indent -= 2
+        if (line.endsWith("}")) indent = max(0, indent - 2)
```

Comment:
Isn't this going to potentially hide errors? What about throwing the input string instead? That'll make troubleshooting easier next time this happens?

Reply:
Yeah, not great. Going to close this PR